src/main/java/com/wcc/platform/domain/cms/pages/aboutUs/CelebrateHerPage.java

```diff
@@ -0,0 +1,16 @@
+package com.wcc.platform.domain.cms.pages.aboutUs;
+
+import com.wcc.platform.domain.cms.pages.Page;
+import com.wcc.platform.domain.platform.Member;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+/**
+ * CMS Celebrate Her page.
+ *
+ * @param page Page details as title and images
+ * @param members all details
+ */
+public record CelebrateHerPage(
+    @NotNull Page page,
+    @NotNull List<Member> members) {}
```

Comment:
I think would be better to create another model for that content, because it should be 4 attributes more or less.